workbench-core/datasets/src/dataSetService.test.ts

```diff
@@ -56,8 +58,31 @@ describe('DataSetService', () => {
         secretAccessKey: 'fakeSecret'
       }
     });
+    const mockAuthService = {
```

Comment:
Is there a type we can add here so that if things change in the future we can still know what is required?